test/Gen.hs

```diff
@@ -25,12 +26,9 @@ module Gen
 , identity
   -- * Handlers
 , Run(..)
-, type RunL
-, pattern RunL
-, type RunR
-, pattern RunR
-, type RunC
-, pattern RunC
```

Comment:
Generalizing all of the tests over the input functor obviated the need for these.